src/main/java/com/google/error_analyzer/backend/DaoInterface.java

```diff
@@ -0,0 +1,52 @@
+/**Copyright 2019 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.*/
+package com.google.error_analyzer.backend;
+
+import org.elasticsearch.search.SearchHit;
+import java.io.IOException;
+import java.lang.*;
+import java.util.*;
+
+public interface DaoInterface {
```

Comment:
Let's try to keep here only the functions interacting with database.
Others we can move to a new LogDaoHelper class.

Reply:
Also, functions in LogDaoHelper can have unit tests.